src/main/kotlin/org/rust/ide/actions/RsFmtFileAction.kt

```diff
@@ -48,7 +48,12 @@ class RsFmtFileAction : DumbAwareAction() {
         try {
             cargo.reformatFile(module, file)
         } catch (e: ExecutionException) {
-            project.showBalloon(e.message ?: "", NotificationType.ERROR)
+            // #1131 - Check if we get a `no such subcommand: fmt` and let the user know to install fmt
+            if ("no such subcommand: fmt" in e.message) {
```

Comment:
Hm, looks like `e.message` is nullable on at least one version of IJ. I think we can do

```
val message = e.message ?: ""
```

and then use it in this condition and also in the else branch.

Reply:
I'll after this PR definitely go learn Kotlin! Silly mistakes like this shouldn't be happening.